websitev2/sidebars.js

```diff
@@ -59,6 +59,10 @@ module.exports = {
             'advanced/api_access',
             'advanced/tags',
             'advanced/enterprise-authentication',
+            'advanced/sso-saml-keycloak',
```

Comment:
I was thinking we did not want the keycloack in the sidebar itself, to reduce the number of items. What do you think?

Reply:
I don't think it hurts. All SSO docs are clearly marked with SSO and easily distinguishable from the rest of the advanced items. With the sidebars being collapsable it doesn't really make too much noise in my opinion.